app/serializers/service_provider_serializer.rb

```diff
@@ -48,7 +48,13 @@ def certs
   end
 
   def pkce
-    object.openid_connect_pkce?
+    if ['urn:gov:gsa:sp:sso:frb_stl:scs', 'urn:gov:gsa:sp:sso:frb_stl:ecs'].include?(object.issuer)
```

Comment:
would it be worth putting these in a config instead of hardcoding in source?

Reply:
Yep, good call, I forgot we also have config here!